codalab/lib/codalab_manager.py

```diff
@@ -223,6 +223,11 @@ def init_config(self, dry_run=False):
             config['server']['engine_url'] = "sqlite:///{}".format(sqlite_db_path)
             print "Using SQLite database at: {}".format(sqlite_db_path)
 
+        # Generate secret key
+        config['server']['secret_key'] = get_random_string(
+            48, "=+/abcdefghijklmnopqrstuvwxyz"
+                "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
```

Comment:
This is a more general problem of updating the config file.  Perhaps there should be a separate cl command that reads in the config.json, adds fields that don't exist, and writes it back?